pkg/reconcilermanager/controllers/rootsync_controller.go

```diff
@@ -305,6 +311,34 @@ func (r *RootSyncReconciler) setup(ctx context.Context, reconcilerRef, rsRef typ
 	return nil
 }
 
+// teardown performs the following teardown steps:
+// - Set the Reconciling condition with the Teardown reason (in-progress)
+// - Call deleteManagedObjects and update the Stalled condition if it errors
+// - Update the Reconciling condition with the Teardown reason (complete)
```

Comment:
Swap these two lines to match the implementation.

Reply:
fixed.